src/components/common/ColonyActions/ActionDetailsPage/DefaultAction/DefaultAction.tsx

```diff
@@ -21,17 +20,18 @@ interface DefaultActionProps {
 const DefaultAction = ({ item }: DefaultActionProps) => {
   const { colony } = useColonyContext();
 
+  const {
+    enabledExtensions: { isVotingReputationEnabled },
+  } = useEnabledExtensions();
+
   if (!colony) {
     return null;
   }
 
-  const isVotingExtensionEnabled = false;
-  const motionStyles = MOTION_TAG_MAP[MotionState.Forced];
-
   return (
     <div className={styles.main}>
       {/* {isMobile && <ColonyHomeInfo showNavigation isMobile />} */}
-      {isVotingExtensionEnabled && <MotionTag motionStyles={motionStyles} />}
+      {isVotingReputationEnabled && <ForcedTag />}
```

Comment:
An action that goes through when the voting reputation extn is enabled is "forced". 

But should it still show up as forced if the voting rep extn is currently enabled, but the action itself went through when the voting rep extn was not enabled? @arrenv